signal_strength_tracker.py

```diff
@@ -0,0 +1,48 @@
+"""
+Logs LTE signal strength into a file.
+"""
+import datetime
+import pathlib
+import time
+
+import serial
+
+
+# Parameters
+SERIAL_PORT = "/dev/ttyUSB3"  # Change this to match your modem's serial port
+BAUDRATE = 115200  # Change this to match your modem's baud rate
+AT_COMMAND = b"AT+CSQ\r\n"  # Command to run on modem
+OUTPUT_LOG_FILE_PATH = pathlib.Path(f"/media/warg/E3F9-7EE3/signal_log_{int(time.time())}.txt")
+PERIOD = 3
+SETUP_TIME = 20
+
+if __name__ == "__main__":
+    # Wait 20 seconds becasue the RPi has a setting where it doesn't boot until 20 seconds after power
```

Comment:
Is it the Raspberry Pi itself that doesn't boot for 20 seconds? If the Raspberry Pi isn't booted how is the program supposed to run? Is it something else that isn't booting immediately?